docs/arbos/Gas.md

```diff
@@ -4,7 +4,7 @@ There are two parties a user pays when submitting a tx:
 - the poster, if reimbursable, for L1 resources such as the L1 calldata needed to post the tx
 - the network fee account for L2 resources, which include the computation, storage, and other burdens L2 nodes must bare to service the tx
 
-Though subject to change when batch-compression pricing is fully implemented, [the L1 component](ArbOS.md#l1pricingstate) of a tx's fee consists of the approximate cost of posting the tx's calldata to the L1 inbox. This best-effort estimate is computed using ArbOS's running L1 gas-price estimate and the poster's self-reported compressibility-discount ratio. These [charges are dropped][drop_l1_link] if the poster is not the user's preferred aggregator, which the user may set using [`ArbAggregator`](Precompiles.md#ArbAggregator)'s [`SetPreferredAggregator`](Precompiles.md#ArbAggregator) precompile method.
+The L1 component is the product of the tx's estimated calldata size and the L2's view of the L1 data price, a value which dynamically adjusts over time to ensure the batch-poster is ultimately fairly compensated. For details, see [ L1 Pricing](ArbOS.md#l1pricingstate).
```

Comment:
It's the estimated contribution to the batch's size, which we compute using Brotli compression on the individual transaction by itself